recipes/kibana4.rb

```diff
@@ -52,3 +52,12 @@
   )
   notifies :restart, 'service[kibana]'
 end
+
+# Install plugins
+node['kibana']['plugins'].each do |plugin|
+  execute "install #{plugin[:name]}" do
+    command "/usr/local/kibana-#{node['kibana']['kibana4_version']}/bin/kibana plugin --install #{plugin[:url]}"
+    not_if { ::Dir.exists?("/usr/local/kibana-#{node['kibana']['kibana4_version']}/installedPlugins/#{plugin[:name]}") }
```

Comment:
sorry for not following up on this.. i didn't get any notifications from Github. i'll push some updates.